src/interfaces/state-interface.ts

```diff
@@ -1,14 +1,17 @@
 import { NewRequestIF } from '@/interfaces/new-request-interface'
+import { StaffPaymentIF } from '@/interfaces'
 
 export interface StateIF {
   stateModel: StateModelIF
 }
 
 export interface StateModelIF {
   common: {
-    currentJsDate: Date
-  },
+    currentJsDate: Date,
+    keycloakRoles: Array<string>
+  }
   newRequestModel: NewRequestIF
-  errorModel?: {},
+  staffPayment: StaffPaymentIF
```

Comment:
This is where staff payment data is stored. It is not saved to the filing (NR) yet -- this is future work that will be dictated by the spike ticket and API ticket in this story (6288).